Sources/Features/Metrics/Model/Event.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+class Event: Encodable {
```

Comment:
Please consider using immutable data types when there is no need to mutable the properties of a given object. Also, I'd suggest using `struct`s for data objects rather than `class`es. `struct`s are passed by value, so they are safe in a multithreading environment. See more: https://docs.swift.org/swift-book/LanguageGuide/ClassesAndStructures.html

Reply:
Will do, thank you!

Reply:
Will do, thank you!